src/Api/HL/Controller/AdministrationController.php

```diff
@@ -510,6 +556,42 @@ public function getUserByUsername(Request $request): Response
         return Search::getOneBySchema($this->getKnownSchema('User'), $request->getAttributes(), $request->getParameters(), 'username');
     }
 
+    #[Route(path: '/User/{id}/Picture', methods: ['GET'])]
+    #[Doc\Route(
+        description: 'Get the picture for the current user'
+    )]
+    public function getUserPictureByID(Request $request): Response
+    {
+        global $DB;
+        $it = $DB->request([
+            'SELECT' => ['name', 'picture'],
+            'FROM' => User::getTable(),
+            'WHERE' => [
+                'id' => $request->getAttribute('id'),
+            ],
+        ]);
+        $data = $it->current();
+        return $this->getUserPictureResponse($data['name'], $data['picture']);
+    }
+
+    #[Route(path: '/User/username/{username}/Picture', methods: ['GET'])]
```

Comment:
```suggestion
    #[Route(path: '/User/{username}/Picture', methods: ['GET'])]
```

Reply:
It matches the other routes for interacting with users by their username. It seems unnecessary, but there isn't anything that I know of that stops you from having a user with a numeric username so there would be a path requirement overlap where it would match both paths.